tools/ModuleGenerator/libraries/libscevideoout.h

```diff
@@ -0,0 +1,159 @@
+int PS4_SYSV_ABI sceVideoOutSetGamutMetadata_();
+int PS4_SYSV_ABI sceVideoOutSetToneMap_();
+int PS4_SYSV_ABI sceVideoOutSetWindowModeMargins();
+int PS4_SYSV_ABI sceVideoOutSubmitChangeBufferAttribute();
+int PS4_SYSV_ABI sceVideoOutSubmitEopFlip();
+int PS4_SYSV_ABI sceVideoOutSubmitFlip();
+int PS4_SYSV_ABI sceVideoOutSysAddSetModeEvent();
+int PS4_SYSV_ABI sceVideoOutSysAddVblankEvent();
+int PS4_SYSV_ABI sceVideoOutSysClose();
+int PS4_SYSV_ABI sceVideoOutSysConfigureOutputMode_();
+int PS4_SYSV_ABI sceVideoOutSysCursorDisable();
+int PS4_SYSV_ABI sceVideoOutSysCursorEnable();
+int PS4_SYSV_ABI sceVideoOutSysCursorIsUpdatePending();
+int PS4_SYSV_ABI sceVideoOutSysCursorObtain();
+int PS4_SYSV_ABI sceVideoOutSysCursorRelease();
+int PS4_SYSV_ABI sceVideoOutSysCursorSetPosition();
+int PS4_SYSV_ABI sceVideoOutSysDeleteVblankEvent();
+int PS4_SYSV_ABI sceVideoOutSysGetColorConversionStatus();
+int PS4_SYSV_ABI sceVideoOutSysGetCurrentOutputMode_();
+int PS4_SYSV_ABI sceVideoOutSysGetDeviceCapabilityInfoByBusSpecifier_();
+int PS4_SYSV_ABI sceVideoOutSysGetFlipStatus();
+int PS4_SYSV_ABI sceVideoOutSysGetMonitorInfo_();
+int PS4_SYSV_ABI sceVideoOutSysGetPortStatusInfo_();
+int PS4_SYSV_ABI sceVideoOutSysGetPortStatusInfoByBusSpecifier_();
+int PS4_SYSV_ABI sceVideoOutSysGetResolutionStatus();
+int PS4_SYSV_ABI sceVideoOutSysGetVblankStatus();
+int PS4_SYSV_ABI sceVideoOutSysIsSupportedByMonitorInfo_();
+int PS4_SYSV_ABI sceVideoOutSysReadCrc();
+int PS4_SYSV_ABI sceVideoOutSysReadCrc64();
+int PS4_SYSV_ABI sceVideoOutSysResetAtGpuReset();
+int PS4_SYSV_ABI sceVideoOutSysResetZoomBuffers();
+int PS4_SYSV_ABI sceVideoOutSysSetBlackLevel();
+int PS4_SYSV_ABI sceVideoOutSysSetDcePipeMode();
+int PS4_SYSV_ABI sceVideoOutSysSetDimmer();
+int PS4_SYSV_ABI sceVideoOutSysSetDisplayParameters();
+int PS4_SYSV_ABI sceVideoOutSysSetHdrScopeCallbacks();
+int PS4_SYSV_ABI sceVideoOutSysSetInvertedColors();
+int PS4_SYSV_ABI sceVideoOutSysSetOutputCsc();
+int PS4_SYSV_ABI sceVideoOutSysSetOverscanRatio();
+int PS4_SYSV_ABI sceVideoOutSysSetRedirectFlip();
+int PS4_SYSV_ABI sceVideoOutSysSetZoomBuffers();
+int PS4_SYSV_ABI sceVideoOutSysSubmitSubWindowLayout();
+int PS4_SYSV_ABI sceVideoOutSysUpdateDisplayParameter_();
+int PS4_SYSV_ABI sceVideoOutSysUpdatePrivilege();
+int PS4_SYSV_ABI sceVideoOutSysUpdateRenderingMode();
+int PS4_SYSV_ABI sceVideoOutSysUpdateScalerParameters();
+int PS4_SYSV_ABI sceVideoOutUnlockBuffer();
+int PS4_SYSV_ABI sceVideoOutUnregisterBufferAttribute();
+int PS4_SYSV_ABI sceVideoOutUnregisterBuffers();
+int PS4_SYSV_ABI sceVideoOutWaitVblank();
+int PS4_SYSV_ABI Func_Ek7XQBAkFYA();
+int PS4_SYSV_ABI Func_JDjWoYVTvdk();
+int PS4_SYSV_ABI Func_M4lTPTxoeN0();
+int PS4_SYSV_ABI Func_P5a32OulZQk();
+int PS4_SYSV_ABI Func_lMkkEv4aaHY();
+int PS4_SYSV_ABI Func_oXrnSxFki+c();
+int PS4_SYSV_ABI Func_vXMpe7Murfc();
+int PS4_SYSV_ABI Func_wKJpFhCTrX0();
+int PS4_SYSV_ABI Func_wZI4fmUJMlw();
+int PS4_SYSV_ABI Func_-5AIEU9+11g();
```

Comment:
Αυτό δεν είναι valid ονομα

Reply:
auto einai to encode id ti na kano enoas oti den tha kanei compile?

Reply:
ok to allaksa se hex_id se auta pou einai unknown gia na min iparxei problima